src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -48,8 +51,9 @@ private enum Flags : byte
 #nullable disable
         private readonly TypeSymbol _explicitInterfaceType;
         private ImmutableArray<PropertySymbol> _lazyExplicitInterfaceImplementations;
-        private readonly Flags _propertyFlags;
+        private protected Flags _propertyFlags;
         private readonly RefKind _refKind;
+        private readonly bool _setHasImplementation;
```

Comment:
>_setHasImplementation [](http://example.com/codeflow?start=30&length=21)

I do not think this flag belongs on a property. If we need it, an accessor symbol feels as a more appropriate container. But don't just go making a change to move it. Let me finish the review pass.